book/team.yaml

```
@@ -510,4 +510,26 @@ people:
         icon_pack: fab
         link: https://github.com/mlwrzesien
     user_groups:
-      - Tutorial Lead
\ No newline at end of file
+      - Tutorial Lead
+  # =============
+  - title: Xiaolan Xu	
+    avatar: https://avatars.githubusercontent.com/u/7988251?v=4
+    role: Tutorial Lead
+    organizations:
+    - name: Jet Propulsion Lab
+    - name: Radar Science
```

```suggestion
    organizations:
      - name: Jet Propulsion Lab
      - name: Radar Science
```
make sure indents match levels for previous entries